tests/integration/definition-local-macro-boundary.test.ts

```
@@ -0,0 +1,284 @@
+
+describe('Go-to-definition - local macro do/run boundary', () => {
+    let test_temp_dir: string;
+    let pipeline: ReturnType<typeof build_pipeline>;
+
+    beforeEach(() => {
+        test_temp_dir = mkdtempSync(join(tmpdir(), 'def-local-boundary-'));
+        pipeline = build_pipeline();
+    });
+
+    afterEach(() => {
+        try { pipeline?.scope_resolver?.dispose(); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)